src/components/WwwFrame/TheHeader.vue

```diff
@@ -396,6 +425,11 @@ export default {
 			}
 		}
 	},
+	mounted() {
+		if ('requestIdleCallback' in window) {
+			requestIdleCallback(this.loadLendInfo); // preload the Lend dropdown if we're idle
```

Comment:
This prevents the loading spinner on the Lend dropdown if your browser supports requestIdleCallback (no Safari or IE11).

Reply:
This could be a nice enhancement for those staying within the Ui app (the data would be in the apollo cache) but will likely result in a pretty big request for unnecessary data coming into the page with every refresh.

I would prefer this feature be worked on in a separate PR.

Once it's separated out we'll also need to add a mechanism to ensure this only get's called 1 time. They use a task queue here (https://developer.mozilla.org/en-US/docs/Web/API/Background_Tasks_API) but that is prob overkill for this...

Reply:
Thank you let's pick this up at another time...and maybe do a little checking on the performance of this request.